python_tests/pytest_ops.py

```diff
@@ -24,6 +24,18 @@ def is_tensor(a):
     return isinstance(a, torch.Tensor)
 
 
+def convert_scalar_dtype(a: Union[int, float, complex, bool]):
+    if type(a) is int:
+        return DataType.Int
+    elif type(a) is float:
+        return DataType.Double
+    elif type(a) is complex:
+        return DataType.ComplexDouble
+    elif type(a) is bool:
+        return DataType.Bool
+    return None
```

Comment:
This seems like something that is useful outside of testing. Perhaps it belongs in `nvfuser/utils.py` with a name similar to https://github.com/NVIDIA/Fuser/blob/main/nvfuser/pytorch_utils.py#L30 like `python_type_to_nvfuser_dtype`